stories/message-page/message-page.stories.js

```diff
@@ -0,0 +1,212 @@
+import '../../dist/message-page.css';
+
+export default {
+    title: 'Components/Message Page',
+    parameters: {
+        description: `Message page gives feedback to the user when an app or list is empty, or when an error has occurred. The text and icon of the Message Page change depending on the use case, the layout stays the same.
+        
+The message page can be used in the following situations:
+
+- Filtering with no results
+- Searching with no results
+- Empty app
+- Too many items
+- Item saved as a tile that is not longer available
+- Error
+
+## Responsiveness
+The size of the message page adjusts to fit the available space.
+
+## Guidelines
+
+- The icon in the message page is mandatory. It can vary depending on the message. The default icon is <code class="docs-code">.sap-icon--documents</code>.
+- The title is also mandatory. Add a description with further details if needed. 
+- The message should be as concise as possible.
+        `,
+        tags: ['f3', 'a11y', 'theme', 'development']
+    }
+};
+
+const messageBoxHeight = 800;
+
+export const filter = () => `
+<div style="height: 600px; width: 100%;">
+    <div class="fd-message-page">
+        <div class="fd-message-page__container">
+            <div class="fd-message-page__icon-container">
+                <span class="sap-icon sap-icon--filter fd-message-page__icon"></span>
+            </div>
+            <div class="fd-message-page__content">
+                <div class="fd-message-page__title">
+                    No matching items found.
+                </div>
+                <div class="fd-message-page__subtitle">
+                    Check the filter settings.
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+`;
+
+filter.storyName = 'Message Page with Filter';
+filter.parameters = {
+    docs: {
+        iframeHeight: messageBoxHeight,
+        storyDescription: `Use case: The user has set a filter and there are no results.
```

Comment:
"Use the filter message page for when a user has set a filter and there are no results. This message then prompts the user to remove or re-evaluate the filter they set for the page."